src/app/tests/suites/certification/Test_TC_LVL_4_1.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 24.4.1. [TC-LVL-4.1] Step Verification (DUT as Server)
+
+config:
+    cluster: "Level Control"
+    endpoint: 1
+
+tests:
+    - label: "Sending on command"
+      cluster: "On/Off"
+      command: "on"
+
+    - label: "Precondition: DUT level is set to 0x80"
+      command: "Step"
+      arguments:
+          values:
+              - name: "stepMode"
+                value: 0
+              - name: "stepSize"
+                value: 128
+              - name: "transitionTime"
+                value: 20
+              - name: "optionMask"
+                value: 0
+              - name: "optionOverride"
+                value: 0
+
+    - label: "Wait 3000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 3000
+
+    - label: "Reads current level attribute from DUT"
+      command: "readAttribute"
+      attribute: "current level"
```

Comment:
CamelCase

Reply:
Nothing to be done currently, Test Script are passing and naming conventions are as per SDK.